### PR TITLE
feat: handle-account-grant-managed-task

### PR DESCRIPTION
**Pull request by vince5678**
Enable  account grants for EXECUTE MANAGED TASK

## Test Plan

* [x] acceptance tests

* [ ] …

## References

* https://docs.snowflake.com/en/sql-reference/sql/create-task.html#usage-notes
*

### CODE REVIEW

File: pkg/resources/account_grant_acceptance_test.go [hunk 1]
```diff
@@ -37,3 +37,33 @@ resource "snowflake_account_grant" "test" {
 }
 `, role)
 }
+
+func TestAccAccountGrant_grant_manged_task(t *testing.T) {
```
Comment (alldoami) [inline pkg/resources/account_grant_acceptance_test.go:41]: spelling

File: pkg/resources/account_grant_acceptance_test.go [hunk 2]
```diff
@@ -37,3 +37,33 @@ resource "snowflake_account_grant" "test" {
...
+
+func accountGrantManagedTaskConfig(role string) string {
+	return fmt.Sprintf(`
+
+resource "snowflake_role" "test" {
+  name = "%v"
+}
+
+resource "snowflake_account_grant" "test" {
+  roles          = [snowflake_role.test.name]
```
Comment (alldoami) [inline pkg/resources/account_grant_acceptance_test.go:65]: nit: spacing

**Review by alldoami — COMMENTED** [review]
thanks for adding this! added some comments

File: pkg/resources/account_grant_acceptance_test.go (hunk 2 above)
Comment (vince5678) [reply pkg/resources/account_grant_acceptance_test.go]: updated

File: pkg/resources/account_grant_acceptance_test.go (hunk 1 above)
Comment (vince5678) [reply pkg/resources/account_grant_acceptance_test.go]: updated to TestAcc_AccountGrantManagedTask

**Review by alldoami — APPROVED** [review]

**Pull request merged**